chainstate/tx-verifier/src/transaction_verifier/mod.rs

```diff
@@ -582,6 +542,87 @@ impl<'a, S: TransactionVerifierStorageRef, U: UtxosView> TransactionVerifier<'a,
         }
     }
 
+    pub fn connect_pos_accounting_outputs(
```

Comment:
I don't think this should be pub

Reply:
There might be a misunderstanding on what this does. If I understand correctly, this should be called when connecting a regular transaction.

Same applies for the disconnect function.